src/products/createProducts/AddProduct.jsx

```diff
@@ -0,0 +1,30 @@
+import Button from "@mui/material/Button";
+import ProductDetails from "./ProductDetails";
+
+export default function AddProduct({ openModal, setOpenModal }) {
+  const handleAddProduct = () => {
+    setOpenModal(true);
+  };
+
+  return (
+    <>
+      <Button variant="outlined" color="inherit" onClick={handleAddProduct}>
+        Add Product
+      </Button>
+      {openModal && (
+        <ProductDetails
+          openModal={openModal}
+          toAdd={true}
+          setOpenModal={setOpenModal}
+          product={{
```

Comment:
Here as well, create this product above as variable and just pass the entire product variable

Reply:
Okay, noted.